config/api.yml

```diff
@@ -88,6 +88,29 @@
        :post:
        - :name: create
          :identifier: alert_status_action_new
+  :alert_definitions:
+    :description: Alert Definitions
+    :identifier: alert_definition
+    :options:
+    - :collection
+    :verbs: *gpd
+    :klass: MiqAlert
+    :collection_actions:
+      :get:
+      - :name: read
+        :identifier: alert_definitions_show_list
+      :post:
+      - :name: create
+        :identifier: alert_definition_new
```

Comment:
@abellotti Is there anywhere else in the API I can find an example for delete action on the collection level?

adding:
```yaml
:delete:
- :name: delete
  :identifier: alert_definition_delete
```
to the :collection_actions: results in:
```yaml
{
  "error": {
    "kind": "bad_request",
    "message": "Must specify an id for deleting a alert_definitions resource",
    "klass": "Api::BadRequestError"
  }
}
```